src/catala/catala_surface/desugaring.ml

```diff
@@ -394,9 +406,17 @@ let rec translate_expr (scope : Scopelang.Ast.ScopeName.t) (ctxt : Name_resoluti
         | Ast.Aggregate (Ast.AggregateSum Ast.Duration) ->
             make_body (Dcalc.Ast.Add Dcalc.Ast.KDuration)
         | Ast.Aggregate (Ast.AggregateSum _) -> assert false (* should not happen *)
-        | Ast.Aggregate (Ast.AggregateExtremum _) ->
-            Errors.raise_spanned_error "Unsupported feature: minimum and maximum"
-              (Pos.get_position op')
+        | Ast.Aggregate (Ast.AggregateExtremum (max_or_min, t, _)) ->
+            let op_kind =
+              match t with
+              | Ast.Integer -> Dcalc.Ast.KInt
+              | Ast.Decimal -> Dcalc.Ast.KRat
+              | Ast.Money -> Dcalc.Ast.KMoney
+              | Ast.Duration -> Dcalc.Ast.KDuration
+              | _ -> assert false
```

Comment:
This should raise an error message like `Errors.raise_spanned_error (Format.asprintf "It does not make sense to get the maximum of values having the %a type" ...) (Pos.get_position op')